src/handlers.cpp

```diff
@@ -2548,6 +2550,28 @@ void PushProfileTask::update_reg_data()
     _impus = _irs_impus;
   }
 
+  if (_ims_sub_present && is_first_irs())
+  {
+    HTTPCode rc = HTTP_OK;
+    find_impus_to_delete();
+
+    rc = _cfg->sprout_conn->change_associated_identities(_default_public_id,
+                                                         _ims_subscription,
+                                                         this->trail());
+
+    if (rc == HTTP_OK)
+    {
+      TRC_DEBUG("Informed Sprout, return code %d", rc);
```

Comment:
The return code isn't interesting, as you know it must be HTTP_OK to have gone into this branch. I'm not sure this TRC adds anything thinking about it some more - maybe change it to a `if (rc != HTTP_OK)`